website/docs/specification/ConformityCredential.md

```diff
@@ -7,17 +7,651 @@ import Disclaimer from '../\_disclaimer.mdx';
+|social.labour | claims supporting labour rights including fair wages|
+|social.rights | claims supporting human rights and anti-discrimination|
+|social.community | claims supporting local community development|
+|social.safety | claims supporting process and product safety|
+|governance.ethics | claims supporting ethical conduct and corporate governance|
+|governance.compliance | claims supporting regulatory compliance including taxation and community protection|
+|governance.transparency | claims supporting transparency and traceability|
+
+
+### mimeType
+
+```
+Code values for this table can be found here:
+https://mimetype.io/all-types
+```
+
+
+### encryptionMethod
+
+|Name | Value | Description|
+|--- | --- | ---|
+|none |  | no encryption|
+|AES |  | AES standard|
+
+
+### evidenceFormat
+
+ A code describing the format of the conformity evidence
+
+|Name | Description|
+|--- | ---|
+|w3c_vc | A W3C Verifiable Credential|
+|iso_mdl | an ISO 108013 identity credential|
+|document | a binary document for human consumption such as a PDF|
+|website | a reference to an entry on a public website.|
+|other | some other representation|
+
+
+### unitOfMeasure
+
+ UNECE Recommendation 20 Unit of Measure codelist
+
+```
+Code values for this table can be found here:
+https://vocabulary.uncefact.org/UnitMeasureCode
+```
+
+## Sample
+
+```
+{
+  "id": "http://example.com",
+  "assessorLevel": "Self",
+  "assessmentLevel": "GovtApproval",
+  "type": "certification",
+  "description": "string",
+  "scope": {
+    "id": "http://example.com",
+    "name": "string",
+    "trustmark": {
+      "fileHash": "string",
+      "fileLocation": "http://example.com",
+      "fileType": "string",
+      "EncryptionMethod": "none"
+    },
+    "issuingBody": {
+      "identifiers": [
+        {
+          "scheme": "http://example.com",
+          "identiferValue": "string",
+          "identifierURI": "http://example.com",
+          "verificationEvidence": {
+            "format": "w3c_vc",
+            "credentialReference": "http://example.com"
+          }
+        }
+      ],
+      "name": "string"
+    },
+    "dateOfIssue": "2019-08-24"
+  },
+  "issuedBy": {
+    "identifiers": [
+      {
+        "scheme": "http://example.com",
+        "identiferValue": "string",
+        "identifierURI": "http://example.com",
+        "verificationEvidence": {
+          "format": "w3c_vc",
+          "credentialReference": "http://example.com"
+        }
+      }
+    ],
+    "name": "string"
+  },
+  "issuedTo": {
+    "identifiers": [
+      {
+        "scheme": "http://example.com",
+        "identiferValue": "string",
+        "identifierURI": "http://example.com",
+        "verificationEvidence": {
+          "format": "w3c_vc",
+          "credentialReference": "http://example.com"
+        }
+      }
+    ],
+    "name": "string"
+  },
+  "validFrom": "2019-08-24",
+  "validTo": "2019-08-24",
+  "status": "string",
+  "assessments": [
```

Comment:
... and this be `credentialSubject`?

Reply:
I was thinking that, like the DPP model, the entire DPCC is independent of envelope.  So when represented as a VC, the entire structure is contained in credentialSubject.  So the VCDM "issuer" would be the DID of the CAB.  But the "issuedBy" party in the DPCC would be the name and tax registration number of the CAB.  similar with status.  VCDM status is typically a simple revocation status. Whereas the status in the body of the credential can have a more complex lifecycle.  

But it's a fair point to discuss these questions - ie about how standard VCDM properties map to similar properties in the credential subject.  I suspect my simple model of keeping them separate and possibly duplicated is not always right.  But then what is the right model?  "issuer" in VCDM is not an array - but in reality many parties have a collection of identifiers and sometimes it's critical to establish relationships between them (eg this DID is also that tax registration holder). 

I think this question should be a topic for discussion at the next meeting.  Can we let this PR through and then address this question of how envelope maps to content consistently for DPP, DPCC, and TE?

Reply:
@onthebreeze , what I'm reacting against is that you are essentially recreating the VC Data Model (defining issuer, validFrom, etc..), seemingly with the intent to wrap that all into the `credentialSubject` of a VC. That will produce a quite inelegant data graph with a lot of duplication and potential for inconsistencies (`issuer` vs `issuedBy`, `validFrom` vs `validFrom`...). 

Rather, we want to leverage the VCDM. Take the issuer, as you mention, indeed it needs an id, but it can easily be added within a complex object. For example: 

```
  ...
  "issuer": {
    "@type": [
      "Organization"
    ],
    "@id": "did:web:nis.example.com",
    "taxIdentificationNumnber": "DK-12345678",
    "name": "Nis Global Unltd.",
    "street": "Ocean Lane 100",
    "locality": "Paradise Island",
    "country": "DK"
  },
  "credentialSubject": {
    "type": [
      "ConformityStuff"
    ],
    "aClaim": "conforming",
    "anotherClaim": "legit",
    ...
```

Reply:
ok, that makes sense.  The thing that threw me was that all VC examples I see always have "issuer" as a simple property that doesnt work for us.  But making it a complex object is perfect.  

But we need to fix this for the DPP model and TE (traceability events) model as well.